src/main/java/io/dockstore/provision/S3CmdPlugin.java

```diff
@@ -99,10 +99,20 @@ public boolean downloadFrom(String sourcePath, Path destination) {
             sourcePath = sourcePath.replaceFirst("s3cmd", "s3");
             String command = client + " -c " + configLocation + " get " + sourcePath + " " + destination + " --force";
             int exitCode = executeConsoleCommand(command, true);
-            if (exitCode != 0) {
+            switch (exitCode) {
```

Comment:
Please throw in a few comments and link to the source code where you found descriptions of the exit codes for s3cmd